NorthwindCRUD/Controllers/CustomersController.cs

```diff
@@ -76,7 +78,10 @@ public ActionResult<OrderDto[]> GetOrdersByCustomerId(string id)
         }
 
         [HttpPost]
-        [Authorize]
+        [SwaggerResponse(400, "Your inputs do not pass validation!", typeof(Errors), "text/json")]
+        [SwaggerResponse(401, "Not authenticated!", typeof(CustomError), "text/json")]
```

Comment:
```suggestion
        [SwaggerResponse(401, "Unauthorized!", typeof(CustomError), "text/json")]
```

Reply:
Using "Unauthorized" is better as it suggests that authentication has occurred, but authorization is lacking. "Not authenticated" implies that authentication has not taken place or has failed.